apps/ledger-live-desktop/src/renderer/modals/Platform/Exchange/StartExchange/index.tsx

```diff
@@ -4,15 +4,30 @@ import Box from "~/renderer/components/Box";
 import DeviceAction from "~/renderer/components/DeviceAction";
 import { createAction } from "@ledgerhq/live-common/hw/actions/startExchange";
 import startExchange from "@ledgerhq/live-common/exchange/platform/startExchange";
+import { ExchangeType } from "@ledgerhq/live-common/wallet-api/react";
 import connectApp from "@ledgerhq/live-common/hw/connectApp";
+
+export type Data = {
+  onCancel?: (error: Error) => void;
+  exchangeType: ExchangeType;
+  onResult: (startExchangeResult: string) => void;
+};
+export function isStartExchangeData(data: unknown): data is Data {
+  if (data === null || typeof data !== "object") {
+    return false;
+  }
+  return "exchangeType" in data ? true : false;
```

Comment:
```suggestion
  return "exchangeType" in data;
```

same here  in returns already a boolean right?